22HW/src/ui/services/products/addNewProductPage.service.ts

```diff
@@ -0,0 +1,19 @@
+import { IProduct } from "../../../data/types/product.types";
+import { logStep } from "../../../utils/reporter/decorators";
+import addNewProductPage from "../../pages/Products/addNewProduct.page";
+import productsPage from "../../pages/Products/products.page";
+import { SalesPortalPageService } from "../salesPortalPage.service";
+
+class AddNewProductService extends SalesPortalPageService {
+  private addNewProductPage = addNewProductPage;
+  private productsPage = productsPage;
+
+  @logStep("Fill all fields to create new prodact and submit")
+  async fillAndSubmit(product: IProduct) {
```

Comment:
```suggestion
  async create(product: IProduct) {
```

Не нужно в названии несколько действий писать, это путает